content/docs/5.1/deploy/operate/maintain.md

```diff
@@ -0,0 +1,206 @@
+---
+title: TiUP Common Operations
+description: Learn the common operations to operate and maintain a TiKV cluster using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 3
+---
+
+This document describes the following common operations when you operate and maintain a TiKV cluster using TiUP.
+
+- [View the cluster list](./#view-the-cluster-list)
+- [Start the cluster](./#start-the-cluster)
+- [View the cluster status](./#view-the-cluster-status)
+- [Modify the configuration](./#modify-the-configuration)
+- [Rename the cluster](./#rename-the-cluster)
+- [Stop the cluster](./#stop-the-cluster)
+- [Clean up cluster data](./#clean-up-cluster-data)
+- [Destroy the cluster](./#destroy-the-cluster)
+
+## View the cluster list
+
+You can manage multiple TiKV clusters using the TiUP cluster component. When a TiKV cluster is deployed, the cluster appears in the TiUP cluster list.
+
+To view the list, run the following command:
+
+```shell
+tiup cluster list
+```
+
+## Start the cluster
+
+The components in the TiKV cluster are started in the following order (The monitoring component is also started):
```

Comment:
when will the monitoring components start?

Reply:
yes, I will add the missing components.

Reply:
yes, TiUP will start monitoring components after tikv. 
But monitoring components' start order is not a `Must` (Monitoring components can be started before TiKV and PD), so it is not mentioned here.

Reply:
@zz-jason PTAL https://github.com/tikv/website/pull/232/commits/84a0afc3cb715320ecdc30e30340911122c5049e